### PR TITLE
transmission-remote: add page

### PR DESCRIPTION
**Pull request by terminalnode**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

Some thoughts on this PR:
- The trickiest thing to figure out with this command is how to select torrents. The option -t is quite flexible. I tried to illustrate different ways in which -t can be used to select individual torrents, ranges such as torrent 1-10, combinations of the two as well as the option "all".
- If `{{hostname}}` isn't specified it's assumed to be localhost. I'd like to illustrate this somehow since a lot of people will be connecting to a local daemon, but I'm not sure how best to illustrate it without it looking like a mistake in the page.
- The order of the arguments when dealing with `-t` and `-a` is very important. This is made clear with `-t` I think, but less so for `-a`. For example in the first example if -a and -w are flipped the default download directory is changed permanently, but in the example the download directory is only specified for the new torrent.

### CODE REVIEW

**Comment by terminalnode** [comment]
For the record there is a tldr entry on `transmission-cli`, [which has been deprecated](https://github.com/transmission/transmission/commit/950387ab5a443629598f93c057f41150707866ab) in favor of `transmission-remote`:

> Limited to a single torrent at a time, transmission-cli is deprecated and exists primarily to support older hardware dependent upon it. In almost all instances, transmission-remote should be used instead.

**Comment by pxgamer** [comment]
Thanks for the info about `transmission-cli`, shows how long it's been since I've used it.

I wonder if I should add a note to the description in the other page such as:
> This tool has been deprecated, please see `transmission-remote`.

File: pages/common/transmission-remote.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# transmission-remote
+
+> Remote control utility for transmission-daemon and transmission.
+
+- Add a torrent file or magnet link to Transmission and download to a specified directory:
+
+`transmission-remote {{hostname}} -a {{torrent|url}} -w {{/path/to/download/to}}`
```
Comment (pxgamer) [inline pages/common/transmission-remote.md]:
```suggestion
`transmission-remote {{hostname}} -a {{torrent|url}} -w {{path/to/download_directory}}`
```

File: pages/common/transmission-remote.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Change the default download directory:
+
+`transmission-remote {{hostname}} -w {{/path/to/download/to}}`
```
Comment (pxgamer) [inline pages/common/transmission-remote.md]:
```suggestion
`transmission-remote {{hostname}} -w {{path/to/download_directory}}`
```

File: pages/common/transmission-remote.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+`transmission-remote {{hostname}} -t {{1}} --remove -t {{2}} --remove-and-delete`
+
+- Stop all torrents:
+
+`transmission-remote {{hostname}} -t {{all}} --stop`
+
+- Move torrents 1-10 and 15-20 to a new folder (folder will be created if it does not exist):
+
+`transmission-remote {{hostname}} -t {{"1-10,15-20"}} --move {{/new/path}}`
```
Comment (pxgamer) [inline pages/common/transmission-remote.md]:
```suggestion
`transmission-remote {{hostname}} -t {{"1-10,15-20"}} --move {{path/to/new_directory}}`
```

**Comment by terminalnode** [comment]
All requested changes commited. :)

**Comment by terminalnode** [comment]
> Thanks for the info about `transmission-cli`, shows how long it's been since I've used it.
> 
> I wonder if I should add a note to the description in the other page such as:
> 
> > This tool has been deprecated, please see `transmission-remote`.

I think that's a good idea, as a user of the program that's very much the kind of information I'd want there. I think I saw a discussion on another page, can't remember which, where something like that was suggested so I don't think it's unprecedented.

**Comment by pxgamer** [comment]
I'll open a PR later after Endgame. 😋

**Pull request merged**